.yarnrc.yml

```diff
@@ -8,6 +8,15 @@ logFilters:
 
 nodeLinker: node-modules
 
+npmRegistries:
+  "https://npm.pkg.github.com":
+    npmAlwaysAuth: true
+    npmAuthToken: "${GITHUB_PACKAGE_READ_TOKEN-}"
+
+npmScopes:
+  metamask:
+    npmRegistryServer: "${METAMASK_NPM_REGISTRY:-https://registry.yarnpkg.com}"
```

Comment:
This has no impact upon most development environments because `METAMASK_NPM_REGISTRY` won't be set for most users. This `:-` syntax lets us set a default, which I'm setting here to the default Yarn npm registry `https://registry.yarnpkg.com`. Therefore if `METAMASK_NPM_REGISTRY` isn't set, this is a no-op.